modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -518,18 +521,23 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
       txRequestId,
       rShare,
       encryptedSignerShare.share,
-      apiVersion
+      apiVersion,
+      undefined,
```

Comment:
Maybe there's a way to clean these up so they're not all undefined? Maybe we can put `reqId` into a `params` object {} so we don't have to pass in 4 undefined's before?

Reply:
Yea I also don't like all those `undefined`, initially I thought about just having a `params` object with all of the parameters, but that would be breaking. If we put `reqId` into a `params` object, wouldn't we still need to use `undefined` for the previous parameters? Or do you mean putting all the optional parameters into `params`?

Reply:
Yeah, you're right that would be breaking...hmmmmm. I was thinking passing in all those optionals as a param, but in that case for now this is fine